src/api/app/controllers/webui/package_controller.rb

```diff
@@ -484,6 +484,19 @@ def rdiff
     @files = filenames['files']
     @not_full_diff = @files.any? { |file| file[1]['diff'].try(:[], 'shown') }
     @filenames = filenames['filenames']
+
+    # TODO: moved from the old view, needs refactoring
```

Comment:
it was only copied... should it be removed from the old view if it is now in the controller? :thinking:

Reply:
The old view gets anyway removed in a couple of weeks, no need to clean up IMHO...

Reply:
ok